src/components/NoteItem.vue

```diff
@@ -168,6 +190,23 @@ export default {
 				this.actionsOpen = false
 			}
 		},
+		onToggleSharing() {
+			if (window?.OCA?.Files?.Sidebar?.setActiveTab) {
+				emit('toggle-navigation', { open: false })
+				setTimeout(() => {
+					window.dispatchEvent(new Event('resize'))
+				}, 200)
+				window.OCA.Files.Sidebar.setActiveTab('sharing')
+				window.OCA.Files.Sidebar.open(this.note.internalPath.replace('files/', '/'))
```

Comment:
This might be tricky if the path returns files in the middle, but should not be necessary anymore with my pushed commit ;)
```suggestion
				window.OCA.Files.Sidebar.open(this.note.internalPath)
```